src/presenters/admin/migration-error-presenter.php

```diff
@@ -16,30 +17,53 @@
 class Migration_Error_Presenter extends Abstract_Presenter {
 
 	/**
-	 * Presents the warning that your site's content is not fully indexed.
+	 * Holds the migration error.
 	 *
-	 * @return string The warning HTML.
+	 * @param int|false $time    The timestamp.
+	 * @param string    $version The Yoast SEO version.
+	 * @param string    $message The error message.
```

Comment:
I do not think this is valid PHPDoc.
`@param` is meant to document function parameters, not array properties, [according to the official documentation](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.param.pkg.html).
Although I do recognize the need to document the internal structure of the array 🤔.

Reply:
There does not seem to be an official way to document array parameters.

I think it would be better to document this as plain text for the time being.
Since this means that PHPStorm would also pick this up (for example when pressing `F1` on the use of the `$this->migration_error` property). 

In the case of documenting the array keys using `@param` tags, no documentation about them show up in PHPStorm when pressing `F1`.